Software/Development/SemesterProject/App/Include/Constants.h

```diff
@@ -63,14 +63,14 @@
 // Constant component values from interface PCB
 #define R_IN_CURRENT_MEAS           9.1     // Ohm
 #define BIAS_VOLTAGE_OPAMP          0.817   // V
-#define OPAMP_GAIN_CURRENT_MEAS     -1      // []
-#define CURRENT_SENSOR_GAIN         2000    // []
-#define R1_DCLINK_MEAS              21500   // Ohm
-#define R2_DCLINK_MEAS              1000    // Ohm
-#define R3_DCLINK_MEAS              16200   // Ohm
-#define R4_DCLINK_MEAS              10000   // Ohm
-#define R1_CONTROL_SUPPLY_MEAS      8200    // Ohm
-#define R2_CONTROL_SUPPLY_MEAS      1000    // Ohm
+#define OPAMP_GAIN_CURRENT_MEAS     (float)-1      // []
+#define CURRENT_SENSOR_GAIN         (float)2000    // []
+#define R1_DCLINK_MEAS              (float)21500   // Ohm
+#define R2_DCLINK_MEAS              (float)1000    // Ohm
+#define R3_DCLINK_MEAS              (float)16200   // Ohm
+#define R4_DCLINK_MEAS              (float)10000   // Ohm
+#define R1_CONTROL_SUPPLY_MEAS      (float)8200    // Ohm
+#define R2_CONTROL_SUPPLY_MEAS      (float)1000    // Ohm

```

Comment:
Why are al this floats if they are really integers?¿

Reply:
The the output of the equations where i use them are floats, so i cast here instead of in the functions

Reply:
You are better than that aitor